configFunctions.f90

```diff
@@ -91,21 +101,24 @@ subroutine findReactionsWithProductOrReactant( r, chs, arrayLen )
     return
   end subroutine findReactionsWithProductOrReactant
 
+  ! ----------------------------------------------------------------- !
```

Comment:
Are these the width of your screen? You obviously have a different linewidth to me! And I suppose these are just visual aids for you to see the separation between functions? Does 2 blank lines not do this? Also, do you use a 'linter', which colours the keywords, which really helps with visualising the flow? I can't remember your setup.